features/feed.feature

```diff
@@ -35,24 +23,17 @@ Feature: Feed
     And the feed contains "Note1"
     And the feed does not contain "Article1"
 
-  Scenario: Feed only includes posts
```

Comment:
Removed this scenario as it doesn't make sense anymore seeming though a feed can only contain posts